### ISSUE TITLE
Build errors on develop branch

### ISSUE THREAD

**Issue by huckl3b3rry87** [autowarefoundation/autoware_ai#85]

**Comment by amc-nu** [comment]
@huckl3b3rry87 I created this package. But I think this package is not the problem.
What’s your Ubuntu version?

**Comment by amc-nu** [comment]
Found the problem I’ll push a fix.

**Comment by amc-nu** [comment]
@huckl3b3rry87 can you please test autowarefoundation/autoware#1149 and check if your error is fixed?

**Comment by slivingston** [comment]
@amc-nu I was able to reproduce the error in the opening post, but instead of autowarefoundation/autoware#1149, I find that it is sufficient to
```diff
diff --git a/ros/src/sensing/fusion/packages/autoware_camera_lidar_calibrator/CMakeLists.txt b/ros/src/sensing/fusion/packages/autoware_camera_lidar_calibrator/CMakeLists.txt
index 5acebbf2..5ad9a5d5 100644
--- a/ros/src/sensing/fusion/packages/autoware_camera_lidar_calibrator/CMakeLists.txt
+++ b/ros/src/sensing/fusion/packages/autoware_camera_lidar_calibrator/CMakeLists.txt
@@ -17,6 +17,7 @@ find_package(catkin REQUIRED COMPONENTS
 
 find_package(OpenCV REQUIRED)
 find_package(PCL)
+find_package(Qt5Core REQUIRED)
 
 catkin_package(CATKIN_DEPENDS
     cv_bridge
```

**Comment by amc-nu** [comment]
fixed in autowarefoundation/autoware#1149
